graylog2-server/src/main/java/org/graylog2/commands/ServerCommandsProvider.java

```diff
@@ -31,6 +31,7 @@ public void addTopLevelCommandsOrGroups(CliBuilder<CliCommand> builder) {
 
         builder.withCommand(Server.class);
         builder.withCommand(MigrateCmd.class);
+        builder.withCommand(ExampleCommand.class);
```

Comment:
Since we should remove the example files, we shouldn't bind the example command here.